src/content/en/updates/2020/10/http-cache-partitioning.md

```diff
@@ -0,0 +1,261 @@
+iframe from `https://c.example`.
+
+In this case, the image is downloaded from the network because there is no
+resource in the cache that matches the key `https://a.example` +
+`https://c.example` + `https://x.example/doge.png`.
+
+<div class="clearfix"></div>
+
+<figure class="attempt-left">
+  <img src="/web/updates/images/2020/10/http-cache-partitioning-7.png">
+  <figcaption>
+    <b>Cache Key</b>: { <code>https://a.example</code>, <code>https://c.example</code>, <code>https://x.example/doge.png</code> }
+  </figcaption>
+</figure>
+
+What if the domain contains a subdomain or a port number? The user visits
+`https://subdomain.a.example`, which embeds an iframe
+(`https://c.example:8080`), which requests the image.
+
+Because the key is created based on "scheme://eTLD+1", subdomains and port
+numbers are ignored, hence a cache hit occurs.
+
+<div class="clearfix"></div>
+
+<figure class="attempt-left">
+  <img src="/web/updates/images/2020/10/http-cache-partitioning-5.png">
+  <figcaption>
+    <b>Cache Key</b>: { <code>https://a.example</code>, <code>https://c.example</code>, <code>https://x.example/doge.png</code> }
+  </figcaption>
+</figure>
+
+What if the iframe is nested multiple times? The user visits
+`https://a.example`, which embeds an iframe (`https://b.example`), which embeds
+yet another iframe (`https://c.example`), which finally requests the image.
+
+Because the key is taken from the top-frame (`https://a.example`) and the
+immediate frame which loads the resource (`https://c.example`), a cache hit
+occurs.
+
+<div class="clearfix"></div>
+
+## FAQs
+
+### As a web developer, are there any action I should take in response to this change?
+
+This is not a breaking change, but it may impose performance considerations for
+some web services.
+
+For example, those that serve large volumes of highly cacheable resources across
+many sites (such as fonts and popular scripts) may see an increase in their
+traffic. Also, those who consume such services may have an increased reliance on
+them.
+
+(There's a proposal to enable shared libraries in a privacy-preserving way
+called [Web Shared
+Libraries](https://docs.google.com/document/d/1lQykm9HgzkPlaKXwpQ9vNc3m2Eq2hF4TY-Vup5wg4qg/edit#)
+([presentation video](https://www.youtube.com/watch?v=cBY3ZcHifXw)), but it's
+still under consideration.)
+
+### What is the impact of this behavioral change?
+
+The overall cache miss rate increases by about 3.6%, changes to the FCP (First
+Contentful Paint) are modest (~0.3%) and the overall fraction of bytes loaded
+from the network increases by around 4%. You can learn more about the impact on
+performance in [the HTTP cache partitioning
+explainer](https://github.com/shivanigithub/http-cache-partitioning#impact-on-metrics).
+
+### Is this standardized? Do other browsers behave differently?
+
+"HTTP cache partitions" is [standardized in the fetch
+spec](https://fetch.spec.whatwg.org/#http-cache-partitions) though browsers
+behave differently:
```

Comment:
This strongly suggests to me that your article should be on web.dev. Did Kayce turn you down?

Reply:
I and @rowan-m agreed that this is Chrome specific change that doesn't require developer action and other browsers behave differently. @kaycebasques what is your thought?

Reply:
For what its worth, its the fact that this is governed by a spec that makes me think web.dev. I'm not married to the idea.